src/app/ASKS/ask/index.js

```diff
@@ -193,13 +193,14 @@ const Ask = ({
         </HStack>
       </FormControl>
     )
-  if (component === 'checkbox')
+  if (component === 'single_checkbox')
```

Comment:
We could remove this now since we are handling it based on type field. The **component** is still going to be **checkbox** where as the **type** is going to be **single_checkbox**